input_buffer.cc

```diff
@@ -35,11 +35,11 @@
 #include "input_buffer.hh"
 #include <ctype.h>
 #include <errno.h>
-#include <limits.h>
```

Comment:
Is this one not needed?

Reply:
I copied that from @bsdjhb's https://github.com/CTSRD-CHERI/cheribsd/commit/c05532bebd36c84aeebf16b734595c5b443fbde6, and it compiles fine for me so I guess not?